### ISSUE TITLE
scanning in dotnet8 and maui is unacceptable slow

### ISSUE THREAD

**Issue by guido-gtt** [Redth/ZXing.Net.Maui#169]
about 10% of the time detecting barcodes is really bad.. 
We try with different light, moving, holding still, different angles, everything we can think of, but it takes sometimes a full minute before a barcode is detected, sometimes it never gets detected.
Tested on a samsung A13 and A14
Previously, we used zxing on Xamarin, this is so good and fast with our barcodes. Bad light and long distances were not a problem there.

we tried AutoRotate = true, TryHarder = true, Multiple = false ) but nothing helps

If there are some workarounds please let us know. Our app is ready, that's the only thing that's stopping us..

Thank you and best regards
Guido

**Comment by SKIDDOW** [comment]
Yes, It is slow, will take some time to release a better version. They're quite busy. I'm finding a solution for months.

**Comment by guido-gtt** [comment]
I left the zxing library and tried this one, it scans very good but has troubles displaying, I need to set a fixed height else it does not show
https://github.com/afriscic/BarcodeScanning.Native.Maui

**Comment by jfversluis** [comment]
Glad you found something that works for you!
